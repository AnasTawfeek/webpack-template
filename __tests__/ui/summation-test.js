import React from 'react';
import renderer from 'react-test-renderer';
import Summation from '../../web/src/ui/summation';

describe('<Summation>', () => {
  let tree;

  describe('when summing 1 and 2', () => {
    beforeEach(() => {
      tree = renderer.create(<Summation x={ 1 } y={ 2 }/>).toJSON();
    });

    it('should return 3', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
