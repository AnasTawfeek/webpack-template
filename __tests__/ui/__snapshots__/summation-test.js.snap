exports[`<Summation> when summing 1 and 2 should return 3 1`] = `
<div>
  <input
    onChange={[Function bound handleValueXChange]}
    type="number"
    value={1} />
  <span>
    +
  </span>
  <input
    onChange={[Function bound handleValueYChange]}
    type="number"
    value={2} />
  <span>
    =
  </span>
  <input
    readOnly={true}
    type="number"
    value={3} />
</div>
`;
